rpa-home/modules/ROOT/pages/automation-usecase-legacysystemdataextraction.adoc

```diff
@@ -0,0 +1,107 @@
+# Usecase: Extract Data from a Legacy System
+
+Extracting data from legacy systems is a common use case for RPA. 
+
+This example illustrates the following tasks:
+
+* Copy the contents from a PDF file to a text file.
+* Compare strings.
+* Copy a substring from a text file.
+* Find text on a screen using optical character recognition (OCR).
+* Interact with a screen using mouse actions and key strokes.
+* Use activity parameters to transfer data and to determine which path to follow in the model.
+
+## Example Scenario
+
+Imagine you work as an accountant at Mythical University. Mythical University has ordered a travel package for one of their field excursions. You have received an email and saved the attached order confirmation to your computer.
+
+This is what an order confirmation of your travel agency Wonderlust Travel typically looks like:
+
+[[confirmationletter-image]]
+image::usecase-1-wonderlusttravels-orderconfirmation-pdf.png["Letter containing a logo, headers, text and tables. The order ID is contained in a table in a header in the middle of the page. The label says 'Order ID'.", 75%]
+
+You need to search the order ID in your order tracking system to get the corresponding tracking number, which you need to reference in your payment.
+
+Unfortunately your order tracking system is very old and accountants can only access it by typing in and reading from a terminal screen:
+
+[[trackingsystem-allentries-image]]
+image::as400ordertrackingscreen.png["Legacy order tracking system showing green glowing text in a terminal. Above a list of orders, there is an entry field for the order id labeled 'Order-ID' and a button labeled 'Search' next to it.", 75%]
+
+You click the search field, enter the order ID, and click 'Search'.
+
+[[trackingsystem-noentries-image]]
+image::as400ordertrackingscreen-searchresult-ordernotfound.png["Legacy order tracking system shows no search results.", 75%]
+
+The list of orders is empty now. This could either mean that the order is'nt there or that you mistyped the number. Let's assume that the latter is the case. You try again and the list shows the corresponding entry:
```

Comment:
```suggestion
The list of orders is now empty, which means that either the order isn't there or you mistyped the number. Let's assume that the latter is the case. You try again and the list shows the corresponding entry:
```

Reply:
Or you could say "The search results in no orders" or something like that.

Reply:
Is there any way to trim the images so they don't take up so much space with just the black terminal? This will make it easier to scan, too.

Reply:
That's the point to prove ;) Scanning those terminals is not easy....

Reply:
... for a human.

Reply:
As a non-native speaker, I find  "The search results in no orders" difficult to understand because I waited for the verb here at first. Like in  "The search results in no orders decided to go to the beach."